integration-tests/utils/zendesk/createAndApproveTicket.ts

```diff
@@ -0,0 +1,23 @@
+import { makeApproveZendeskTicketRequest } from './approveZendeskTicket'
+import { createZendeskTicket } from './createZendeskTicket'
+import { validRequestData } from '../../constants/requestData'
+
+const createAndApproveTicket = async () => {
+  if (!process.env.FIXED_DATA_REQUEST_DATE) {
+    throw new Error(
+      'Must set FIXED_DATA_REQUEST_DATE when calling this utility'
```

Comment:
Should this be a bit more helpful and say it should be set in `env` variables? I'd personally find this confusing and would still have to open up the file to find the test to see what it meant.

Reply:
I've updated the Readme now. Everything you use here has to be set as a shell environment variable.
In any case, if you look at the `package.json`, it sets these variables anyway, so if you use the facility as now described in the Readme, you shouldn't have any confusion.